static/app/views/starfish/modules/APIModule/queries.js

```diff
@@ -178,3 +178,19 @@ export const getHostStatusBreakdownQuery = ({domain, datetime}) => {
     ORDER BY count DESC
   `;
 };
+
+export const getEndpointsThroughputQuery = ({datetime}) => {
```

Comment:
This also needs to filter down to get spans of the correct endpoint no?

Reply:
I'm not sure if we can do that since I'm doing a `GROUP BY` on `description`, but correct me if I'm wrong here. It seems to work the way it is tho

Reply:
I think you need a where clause like this one 
https://github.com/getsentry/sentry/blob/33bcca2c21a4e30154a195fe85e8af74b4c1b049/static/app/views/starfish/modules/APIModule/queries.js#L37

Reply:
we want the span data to be scoped down to the endpoint we're looking at in the slide out panel

Reply:
Ah yes you're right, thanks! Let me fix that